asv_bench/benchmarks/io/sql.py

```diff
@@ -0,0 +1,84 @@
+import sqlite3
+
+import numpy as np
+import pandas.util.testing as tm
+from pandas import DataFrame, date_range, read_sql_query, read_sql_table
+from sqlalchemy import create_engine
+
+from ..pandas_vb_common import setup  # noqa
+
+
+class SQL(object):
+
+    goal_time = 0.2
+    params = (['sqlalchemy', 'sqlite'],
+              ['float', 'float_with_nan', 'string', 'bool', 'int', 'datetime'])
```

Comment:
I would personally keep this benchmark splitted into two classes where one has those params (for testing types) and the other not. As now this creates a lot of extra benchmarks that are not needed